go.sum

```diff
@@ -1,12 +1,11 @@
-github.com/aws/amazon-kinesis-firehose-for-fluent-bit v0.0.0-20190624030122-c41b42995068 h1:cMoOT6QgDnReGqOcg7+d2YtMpamUhijuubTHGy9dlq8=
-github.com/aws/amazon-kinesis-firehose-for-fluent-bit v0.0.0-20190624030122-c41b42995068/go.mod h1:EGtphTrA5BAr9VKTGmK6P+3/ny4l8wTU2tLjqDyP1F8=
 github.com/aws/amazon-kinesis-firehose-for-fluent-bit v0.0.0-20190723183349-6ca31170fc03 h1:83nR9758gu5543Hvyc+z5Hn4oLiYq00+KhbhoqYpdOM=
 github.com/aws/amazon-kinesis-firehose-for-fluent-bit v0.0.0-20190723183349-6ca31170fc03/go.mod h1:jZjLd+hsaK0oNVotfqbMK9gUsIhXvYEl7Z3tdZTYa54=
 github.com/aws/aws-sdk-go v1.20.6 h1:kmy4Gvdlyez1fV4kw5RYxZzWKVyuHZHgPWeU/YvRsV4=
```

Comment:
We need to merge the firehose PR first. Then this can be removed.

All of the plugins depend on the firehose `plugins` package